feature-detects/es6/symbol.js

```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
+    "name": "Official ECMAScript 6 specification",
+    "href": "https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-constructor"
+  }],
+  "polyfills": ["es6shim"],
```

Comment:
are you sure that polyfill is covering all of symbol too?

Reply:
My mistake, es6shim does not cover Symbol.   
Will add this polyfill: https://github.com/medikoo/es6-symbol.
How do you decide which polyfill implementation is the best for this feature?